frontend/src/scenes/feature-flags/featureFlagLogic.ts

```diff
@@ -416,18 +441,42 @@ export const featureFlagLogic = kea<featureFlagLogicType>([
             },
             saveFeatureFlag: async (updatedFlag: Partial<FeatureFlagType>) => {
                 const { created_at, id, ...flag } = updatedFlag
+
+                let preparedFlag = flag
+                // Add payloads mapped to keys in multivariate
+                if (preparedFlag.filters?.multivariate) {
+                    const newPayload = {}
```

Comment:
Too confusing otherwise 😅 
```suggestion
                    const newPayloads = {}
```